lib/main.coffee

```diff
@@ -5,22 +5,34 @@ module.exports =
       default: ''
       type: 'string'
       description: 'Leave empty to use bundled'
+    lintInlineJavaScript:
+      type: 'boolean'
+      default: false
+      description: 'Lint JavaScript inside `<script>` blocks in HTML or PHP files'
```

Comment:
There is a way to update grammarScopes on the fly. You already have the `provider` variable, just replace it's grammarScopes property.